src/main/webapp/app/exercises/programming/assess/code-editor-tutor-assessment-container.component.ts

```diff
@@ -539,7 +539,6 @@ export class CodeEditorTutorAssessmentContainerComponent implements OnInit, OnDe
         this.unreferencedFeedback = feedbacks.filter((feedbackElement) => feedbackElement.reference == undefined && feedbackElement.type === FeedbackType.MANUAL_UNREFERENCED);
         this.referencedFeedback = feedbacks.filter((feedbackElement) => feedbackElement.reference != undefined && feedbackElement.type === FeedbackType.MANUAL);
         this.onFeedbackLoaded.emit();
-        this.validateFeedback();
```

Comment:
Same question as above, why don't we need this validation anymore? Is it related to `handleFeedbackCorrectionRoundTag`?

Reply:
Similar to the explanation above, besides that there is no call to handleFeedbackCorrectionRoundTag on this class.